cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1876,6 +1878,28 @@ public void setSubscriptionCountry(String country) {
     }
 
     public Set<Notification> getNotifications() {
+        return getNotificationsFromLocal();
+    }
+
+    public void getNotifications(boolean combineWithApi, NotificationsCallbackListener notificationsCallbackListener) {
+        Set<Notification> localNotifications = getNotificationsFromLocal();
+
+        if (combineWithApi) {
+            getNotificationsFromApi(new NotificationFromApiCallbackListener() {
+                @Override
+                public void ready(List<Notification> remoteNotifications) {
+                    List<Notification> notifications = new ArrayList<>();
+                    notifications.addAll(localNotifications);
```

Comment:
We need to merge the results without duplicates, only include `remoteNotifications` which are *not* included in `localNotifications`
Also make sure they are sorted correctly.